src/recommendpolicy/downloadTemplates.go

```
@@ -112,7 +111,10 @@ func DownloadAndUnzipRelease() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	_ = updatePolicyRules(strings.TrimSuffix(resp.Filename, ".zip"))
+	err = updatePolicyRules(strings.TrimSuffix(resp.Filename, ".zip"))
+	if err != nil {
+		log.Error().Msgf("Failed to update policy rules %s", err.Error())
```

Should there be return here with err?

---

I didn't return an error as it wasn't returned before, so at least logging it was required. 
But we should return an error in this case.

Done.